Dockerfile

```diff
@@ -48,6 +48,7 @@ RUN curl -L https://github.com/jmespath/jp/releases/download/${JP_VERSION}/jp-li
 WORKDIR azure-cli
 COPY . /azure-cli
 
+RUN pip install ./privates/azure_mgmt_containerregistry-3.0.0rc8-py2.py3-none-any.whl
```

Comment:
Please remove this before merge

Reply:
@adewaleo,  check out `https://pypi.org/project/azure-mgmt-containerregistry/3.0.0rc9/`, you can consolidate the PR now to finish it

Reply:
Also update the requirements.txt files